packages/collector/test/tracing/database/redis/connect-via/cluster.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+const MAX_TRIES = 50;
+const { delay } = require('../../../../../../core/test/test_util');
+
+// node bin/start-test-containers.js --redis-node-0 --redis-node-1 --redis-node-2
+// docker exec -it 2aaaac7b9112 redis-cli -p 6379 cluster info
+module.exports = async function connect(redis, log, tries = 0) {
+  // NOTE: we cannot run redis cluster on Tekton https://github.com/bitnami/charts/issues/28894
+  // NOTE: we cannot run redis cluster on Docker locally https://github.com/redis/node-redis/issues/2815
+  //
+  // Please set the environment variables to connect to the cloud redis cluster:
+  //    export AZURE_REDIS_CLUSTER=team-nodejs-redis-cluster-tekton.redis.cache.windows.net:6380
+  //    export AZURE_REDIS_CLUSTER_PWD=
```

Comment:
Are these credentials available on 1Password? It would be helpful if you could mention that as well.

Reply:
> Are these credentials available on 1Password? 

Yes sure!